src/binary/raw_binary_reader.rs

```diff
@@ -250,23 +250,24 @@ pub struct CursorState {
     // All of the annotations on values in `parents` and the current value.
     // Having a single, reusable Vec reduces allocations and keeps the size of
     // the EncodedValue type (which is frequently moved) small.
-    annotations: Vec<SymbolId>,
+    annotations: Vec<RawSymbolToken>,
 }
 
 /// Verifies that the current value is of the expected type and that the bytes representing that
 /// value have not yet been consumed from the data source. This macro is called by the
 /// BinaryCursor#read_{typename} methods.
```

Comment:
Over here and some other places in comments there is mention of `BinaryCursor`/ `Cursor` which can now be changed to `RawBinaryReader`/`RawReader`.